tools/compat/modules/types/bundle.md

```diff
@@ -0,0 +1,7 @@
+---
+subfeatures:
```

Comment:
Potentially this could just use the compat schema 1:1, e.g. instead be:

```
---
bundle_as_string:
  description: '...'
  __compat:
    # ...
---
```